examples/LLM/llama2/chat_app/torchserve_server_app.py

```diff
@@ -0,0 +1,200 @@
+import json
+import os
+
+import requests
+import streamlit as st
+
+MODEL_NAME = "llamacpp"
+# App title
+st.set_page_config(page_title="🦙💬 Llama 2 TorchServe Serve")
+
+
+def start_server():
+    os.system("torchserve --start --model-store model_store --ncs")
```

Comment:
for snappier starts you can disable the compression in archiver

Reply:
Hmmm..I am not sure I follow.  When archiver is just passing the path to the weights using the yaml file and not the actual weights, this wouldn't come into picture?